tasks/rke2.yml

```diff
@@ -31,59 +30,59 @@
-    ansible.builtin.get_url:
-      url: "{{ rke2_artifact_url }}/{{ rke2_version }}/sha256sum-{{ rke2_architecture }}.txt"
-      dest: "{{ rke2_artifact_path }}/sha256sum-{{ rke2_architecture }}.txt"
-      force: yes
-      mode: 0640
-      timeout: 30
-  - name: Download RKE2 artifacts and compare with checksums
-    ansible.builtin.get_url:
-      url: "{{ rke2_artifact_url }}/{{ rke2_version }}/{{ item }}"
-      dest: "{{ rke2_artifact_path }}/{{ item }}"
-      mode: 0640
-      checksum: "sha256:{{ rke2_artifact_url }}/{{ rke2_version }}/sha256sum-{{ rke2_architecture }}.txt"
-      timeout: 30
-    with_items: "{{ rke2_artifact | reject('search', 'sha256sum') | list }}"
-  rescue:
-  - name: "Remote downloading failed: Downloading locally and pushing to remote hosts"
-    ansible.builtin.pause: # Slight delay to make sure you know it's gonna happen and have time to cancel
-      seconds: 7
-  - name: "Create {{ rke2_airgap_copy_sourcepath }}"
-    delegate_to: localhost
-    run_once: true
-    ansible.builtin.file:
-      path: "{{ rke2_airgap_copy_sourcepath }}"
-      state: directory
-  - name: Download RKE2 checksum locally
-    delegate_to: localhost
-    run_once: true
-    register: checksum_file
-    ansible.builtin.get_url:
-      url: "{{ rke2_artifact_url }}/{{ rke2_version }}/sha256sum-{{ rke2_architecture }}.txt"
-      dest: "{{ rke2_airgap_copy_sourcepath }}/"
-      force: yes
-      mode: 0640
-      timeout: 30
-  - name: Downloading RKE2 artifacts locally
-    delegate_to: localhost
-    run_once: true
-    ansible.builtin.get_url:
-      force: yes
-      url: "{{ item }}"
-      dest: "{{ rke2_airgap_copy_sourcepath }}/"
-    with_items: "{{ [rke2_artifact_url+'/'+rke2_version+'/'] | product(rke2_artifact) | map('join') | list + [rke2_install_bash_url] }}"
-  - name: Copy local RKE2 files to remote hosts
-    ansible.builtin.copy:
-      src: "{{ rke2_airgap_copy_sourcepath }}/{{ item }}"
-      dest: "{{ rke2_artifact_path }}/{{ item }}"
-      mode: 0640
-    with_items: "{{ rke2_artifacts + ['rke2.sh'] }}"
-  - name: Set RKE2 install script permissions
-    ansible.builtin.file:
-      path: "{{ rke2_artifact_path }}/rke2.sh"
-      mode: 0750
+    - name: Download sha256 checksum file
+      ansible.builtin.get_url:
+        url: "{{ rke2_artifact_url }}/{{ rke2_version }}/sha256sum-{{ rke2_architecture }}.txt"
+        dest: "{{ rke2_artifact_path }}/sha256sum-{{ rke2_architecture }}.txt"
+        force: yes
+        mode: 0640
+        timeout: 30
+    - name: Download RKE2 artifacts and compare with checksums
+      ansible.builtin.get_url:
+        url: "{{ rke2_artifact_url }}/{{ rke2_version }}/{{ item }}"
+        dest: "{{ rke2_artifact_path }}/{{ item }}"
+        mode: 0640
+        checksum: "sha256:{{ rke2_artifact_url }}/{{ rke2_version }}/sha256sum-{{ rke2_architecture }}.txt"
+        timeout: 30
+      with_items: "{{ rke2_artifact | reject('search', 'sha256sum') | list }}"
+    rescue:
```

Comment:
Please fix the indentation

Reply:
Already had it fixed in my other branch and chose the other branch to merge here... Should be good to go now.